### PR TITLE
Five Issues Resolved in Easel

### PR DESCRIPTION
**Pull request by faizhashmiRnS**
## Description
Following of the issues resolved in this PR:

1. #812 

2. #827 

3. #831 

4. #809

5. #824

### CODE REVIEW

**Review by mustafapylons — COMMENTED** [review]
Few small issues, otherwise code is good.

**Review by faddat — APPROVED** [review]

File: easel/lib/screens/custom_widgets/initial_draft_detail_dialog.dart [hunk 1]
```diff
@@ -78,21 +93,50 @@ class _DraftDetailDialogState extends State<_DraftDetailDialog> {
...
 
+  void getProgressStatus(DownloadProgress loadingStatus) {
+    if (loadingStatus.progress == null) {
+      scheduleMicrotask(() {
+        setState(() {
+          count++;
+        });
+      });
+    }
+    if (count == 1) {
```
[the remark this answers is not in the document]
Comment (faddat) [reply easel/lib/screens/custom_widgets/initial_draft_detail_dialog.dart:131]: I'm not sure.  With dart, which has a really strong linter built in, I would support any code that doesn't make the linter angry, if that makes sense?

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Comment by faizhashmiRnS** [comment]
@kjawadDeveloper2 can we please close the 5 issues that mentioned in this PR? If not tested please send that for QA.

**Pull request merged**